requirements/edx/custom.txt

```diff
@@ -22,11 +22,11 @@ git+https://github.com/edx-solutions/xblock-group-project.git@0.1.1#egg=xblock-g
 -e git+https://github.com/mckinseyacademy/xblock-diagnosticfeedback.git@v0.2.4#egg=xblock-diagnostic-feedback==0.2.4
 -e git+https://github.com/open-craft/xblock-group-project-v2.git@0.4.10#egg=xblock-group-project-v2==0.4.10
 -e git+https://github.com/open-craft/xblock-virtualreality.git@v0.1.1#egg=xblock-virtualreality==0.1.1
-git+https://github.com/edx-solutions/api-integration.git@v2.3.14#egg=api-integration==2.3.14
+git+https://github.com/edx-solutions/api-integration.git@agrendalath/revert-revert-MCKIN-8418-MCKIN-8730#egg=api-integration==2.3.15
```

Comment:
Since https://github.com/edx-solutions/api-integration/pull/185 is being merged, we can use the release tag here?

Reply:
I included the branch here to test `api-integration` before merging. I will push a new commit here with a proper version after merging `api-integration`.

Reply:
ah, ok then will thumb up when tests are passed